sdk_v2/pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
```

Comment:
I found out that using Hatch to manage Python project is quite simple and flexible compare to poetry, setup-tools, or Flit.
@droctothorpe I would appreciate your feedback on how I created this config.